test/kubernetes/e2e/tests/manifests/full-envoy-validation-helm.yaml

```diff
@@ -1,7 +1,9 @@
 gateway:
   validation:
     failurePolicy: Fail # For "strict" validation mode, fail the validation if webhook server is not available
-    allowWarnings: false # For "strict" validation mode, webhook will also reject warnings
+    allowWarnings: false
```

Comment:
Any reason this is false? I imagined that we could test the fullEnvoyValidation even with allowWarnings=true

Reply:
We could, this doesnt change the option just removes the comment